build.gradle

```diff
@@ -148,6 +236,17 @@ configure(subprojects.findAll {
             rename ".*", "pom.xml"
         }
     }
+    // some shortcut tasks
+    task publishSdaIntern {
```

Comment:
We can keep this for now, but should remove it soon after we verified that the maven central upload works. We don't the additional upload.